flygym/simulation.py

```diff
@@ -218,10 +218,33 @@ def step(
         obs, reward, terminated, truncated, info = {}, {}, {}, {}, {}
 
         for fly in self.flies:
-            k = fly.name
-            obs[k], reward[k], terminated[k], truncated[k], info[k] = fly.post_step(
-                self
-            )
+            key = fly.name
```

Comment:
@tkclam : FYI, I think at some point you renamed `key` to `k` but didn't change the code below. I changed it back to `key`.